torch/utils/cpp_extension.py

```diff
@@ -600,6 +659,9 @@ def CUDAExtension(name, sources, *args, **kwargs):
 
     kwargs['language'] = 'c++'
 
+    define_macros = kwargs.get('define_macros', [])
```

Comment:
I don't see this being used, it it necessary? We could also pass define macros like their command line arguments (`-DTHIS_MACRO`) so I'm not convinced that we should add a separate option for this.

Reply:
Removed this unnecessary segment